bijection-hbase/src/test/scala/com/twitter/bijection/hbase/HBaseInjectionsLaws.scala

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.twitter.bijection.hbase
+
+import org.scalatest.{ PropSpec, MustMatchers }
+import org.scalatest.prop.PropertyChecks
+import org.scalacheck.Arbitrary
+
+import com.twitter.bijection.{ CheckProperties, BaseProperties, Injection }
+import org.apache.hadoop.hbase.io.ImmutableBytesWritable
+
+object HBaseInjectionsLaws {
+  implicit val arbitaryImmutableBytesWritable: Arbitrary[ImmutableBytesWritable] =
+    Arbitrary(for {
+      arbByteArray <- Arbitrary.arbitrary[Array[Byte]]
+    } yield new ImmutableBytesWritable(arbByteArray))
+
+  implicit val equivImmutableBytesWritable = new Equiv[ImmutableBytesWritable] {
```

Comment:
You're talking about the `Equiv` impl? I added that earlier to ensure the test methods were picking up the right ones. I will remove it.